.github/workflows/release.yml

```diff
@@ -22,6 +22,15 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: 20
+      - name: Update npm version
+        run: |
+          cd web_ui/frontend
+
+          # Get the current tag and set the package.json version to it
+          npm version ${GITHUB_REF_NAME:1}
```

Comment:
If we bump npm version for frontend, don't we also need to write the updated `package.json` file back to the repo? Otherwise there will be version discrepancy between released artifacts and our codebase.

We should also set `package.json` version in building the production Docker image.